scrapi/events.py

```diff
@@ -49,7 +52,6 @@ def serialize_fluent_data(data):
 # Ues _index here as to not clutter the namespace for kwargs
 def dispatch(_event, status, _index=None, **kwargs):
     if not settings.USE_FLUENTD:
-        logger.warning('Dispatched called but USE_FLUENTD is False')
```

Comment:
was this just too verbose? Maybe bump the log level to info or debug instead of deleting it